modules/effects/src/effects_root_module.ts

```diff
@@ -1,18 +1,22 @@
 import { NgModule, Inject, Optional } from '@angular/core';
-import { StoreModule } from '@ngrx/store';
+import { StoreModule, Store } from '@ngrx/store';
 import { EffectsRunner } from './effects_runner';
 import { EffectSources } from './effect_sources';
 import { ROOT_EFFECTS } from './tokens';
 
+export const ROOT_EFFECTS_INIT = '@ngrx/store/root effects init' as '@ngrx/store/root effects init';
```

Comment:
I would rename the type string to `@ngrx/effects/init`

Reply:
Yep you're right, don't know what I was thinking!